collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -612,6 +611,11 @@ protected void onActivityResult(int requestCode, int resultCode,
             return;
         }
 
+        if (intent == null) {
+            Timber.i("The intent has a null value for requestCode: " + requestCode);
```

Comment:
I'm curious to know what app causes this crash but I'm not sure that there's anything we can actually do about it if we know! I'm fine with either leaving it at `i` or using `w`.

As for moving to Crashlytics, I'd prefer we do it when the integration is more complete, it seems like it hasn't officially been migrated to being the crash log provider yet.